src/Bathymetry.jl

```diff
@@ -116,14 +86,15 @@ Keyword Arguments
                   If `Inf` then no basins are removed. Default: 1.
 """
 function regrid_bathymetry(target_grid;
+                           metadata = nothing,
                            height_above_water = nothing,
                            minimum_depth = 0,
-                           dir = download_bathymetry_cache,
-                           url = etopo_url,
-                           filename = "ETOPO_2022_v1_60s_N90W180_surface.nc",
                            interpolation_passes = 1,
                            major_basins = 1) # Allow an `Inf` number of "lakes"
 
+    if isnothing(metadata)
+        throw(ArgumentError("Metadata must be provided"))
+    end
```

Comment:
if metadata needs to be provided then no need to give it a default `nothing` value upstairs

Reply:
I also prefer to have it as a positional argument, since we have it everywhere as a positional argument